lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -70,6 +70,13 @@
     default: false
     type: bool
     version_added: "2.10"
+  only_tables:
+    description:
+      - dump only tabels of exactly one database, without use database inside the dump
```

Comment:
```suggestion
      - Dump only tabels of exactly one database, without use database inside the dump.
      - Can be used only when I(state=dump).
      - Mutually exclusive with ....
```
because of ```module.fail_json(msg="Multiple databases are not supported with only_tables=yes and  state=dump")```
In general, we need to warn user when this option works, when not